src/app/components/article/MediaArticlesCard.js

```diff
@@ -48,11 +48,10 @@ const MediaArticlesCard = ({ article, team, onAdd }) => {
       key={article.id}
       placement="top"
       title={
-        <FormattedMessage
-          id="articlesSidebarCard.tooltip"
-          defaultMessage="Add article to this media cluster"
-          description="Tooltip message displayed on explainer cards on item page."
-        />
+        <>
+          { article.nodeType === 'FactCheck' && <FormattedMessage id="articlesSidebarCard.factcheckTooltip" defaultMessage="Add Claim & Fact-Check article to this media cluster" description="Tooltip message displayed on explainer cards on item page for fact-check type articles." /> }
+          { article.nodeType === 'Explainer' && <FormattedMessage id="articlesSidebarCard.explainerTooltip" defaultMessage="Add Explainer article to this media cluster" description="Tooltip message displayed on explainer cards on item page for explainer type articles." /> }
+        </>
```

Comment:
The namespace ID for these `<FormattedMessage />`'s should be `mediaArticlesCard.`

Reply:
done